code/modules/plumbing/shower.dm

```diff
@@ -0,0 +1,223 @@
+/obj/machinery/shower
```

Comment:
Sorry, I guess? The parts I changed are everything involving reagents. I'll go through and highlight the particulars in a few but something just came up so it'll take me ~20 minutes at the most, hopefully